Modules/_bisectmodule.c

```diff
@@ -131,7 +130,7 @@ _bisect_insort_right_impl(PyObject *module, PyObject *a, PyObject *x,
             return NULL;
     }
     else {
-        result = _PyObject_CallMethodId(a, &PyId_insert, "nO", index, x);
+        result = PyObject_CallMethod(a, "insert", "nO", index, x);
```

Comment:
Is there a reason this is using `PyObject_CallMethod` (without the underscore)? There seem to be 10 call sites where `_PyObject_CallMethodId` is converted to `PyObject_CallMethod`.